pyshtools/shclasses/shmaggrid.py

```diff
@@ -288,7 +288,7 @@ def plot_theta(self, projection=None, tick_interval=[30, 30],
         cb_triangles : str, optional, default = 'neither'
             Add triangles to the edges of the colorbar for minimum and maximum
             values. Can be 'neither', 'both', 'min', or 'max'.
-        cb_label : str, optional, default = '$B_\\theta$, nT'
```

Comment:
This sort of change won't have any effect as it's just text inside a docstring. You need the `r` on the whole thing, or doubled backslashes.

Reply:
Thanks! I think I caught the remaining errors.